testing/bsg_dmc/Makefile

```diff
@@ -22,6 +22,12 @@ ELABORATE_OPTIONS += -assert svaext
 ELABORATE_OPTIONS += +define+NONSYNTH_TB
 ELABORATE_OPTIONS += +lint=TFIPC-L
 
+# ui_clk period
+# how to use: make elaborate UI_CLK_PRD=<desired clock period value>
+# Note unit is in ps and default is 400MHz as seen below
+UI_CLK_PRD = 2500
+ELABORATE_OPTIONS += +define+UI_CLK_PERIOD=$(UI_CLK_PRD)
```

Comment:
Thanks @akashnt4 ! The manual UI clock tuning is working.

Alternatively, could we "sweep" a handful of preset frequencies automatically? This will be extremely helpful for future developers / users that are not super familiar with the dmc (I bet most people would simply run `make` without manually sweep any clock frequencies..)

Here is an example of frequency sweeping Makefile: https://github.com/bespoke-silicon-group/basejump_stl/blob/master/testing/bsg_dataflow/bsg_parallel_in_serial_out/Makefile. The idea is to create multiple sub-directories for different frequencies, and run simulations in parallel.

Reply:
done

Reply:
Thanks! Could we retain the log files after running, so that we could grep and check the passes / fails of the regression?